reflex/vars.py

```diff
@@ -87,6 +87,15 @@
     "deps": "_deps",
 }
 
+PYTHON_JS_TYPE_MAP = {
+    (int, float): "number",
+    (str,): "string",
+    (bool,): "boolean",
+    (list, tuple): "Array",
+    (dict,): "Object",
+    (None,): "undefined",
```

Comment:
suggest adding a test for the `None` case. i was under the impression that in most circumstances a `None` value mapped to JS `null` rather than `undefined`, but I haven't tested it personally.